site/content/usage/experimental/01-gitops.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "GitOps"
+weight: 10
+url: usage/experimental/gitops-flux
+---
+
+## GitOps
+
+[GitOps][gitops] is a way to do Kubernetes cluster management and application delivery. It works by using Git as a single source
+of truth for declarative infrastructure and applications. With Git at the center of your delivery pipelines, developers
+can make pull requests to accelerate and simplify application deployments and operations tasks to Kubernetes.
+
+`eksctl` provides an easy way to set up gitops in an existing cluster with the `eksctl install flux` command.
+
+[gitops]: https://www.weave.works/technologies/gitops/
+
+
+### Installing Flux
+
+> This is an experimental feature. To enable it, set the environment variable `EKSCTL_EXPERIMENTAL=true`.
```

Comment:
I'd add `**NOTE:**` and clarify that command name, flags and behaviour may change.